plugins/basic/nexus-repository-maven/src/main/java/org/sonatype/nexus/repository/maven/internal/maven2/Maven2Format.java

```diff
@@ -41,6 +44,18 @@
   public static final String METADATA_CONTENT_TYPE = ContentTypes.TEXT_XML;
 
   /**
+   * {@link DateTimeFormatter} for dotted timestamps used in Maven2 repository metadata.
+   */
+  public static final DateTimeFormatter METADATA_DOTTED_TIMESTAMP = DateTimeFormat.forPattern("YYYYMMdd.HHmmss")
+      .withZoneUTC();
```

Comment:
minor: I usually prefer to lock down the locale using `.withLocale(Locale.ENGLISH)` just in case